bcgov/cas-registration
# chore: add facilities page, add BCeID login

card: https://github.com/bcgov/cas-reporting/issues/123

Adding a placeholder page to fetch and display operations and facilities for the user viewing the page. If a user without operators is signed in, the space after ```operators:``` will be blank.

To test:

1. navigate to ```/bc_obps/``` in the terminal and follow the instructions in ```/docs/developer-environment-setup.md/``` to get it running
2. in another terminal run ```yarn dev``` from ```/client/```, open on localhost:3000 and sign in using BCeID. Go through first time setup with one of the accounts in the 1password (bc-cas-dev etc)
3.  in another terminal access the django shell by navigating to ```/bc_obps/``` and running ```python manage.py shell```
4. in the django shell, modify the ```app_role``` value of your user from before. The commands I used to do this are:

from registration.models import User
from registration.models import AppRole
my_user = User.objects.get(first_name=’cas-dev-three’)
cas_admin_role = ppRole.objects.get(role_name=’cas-admin’)
my_user.app_role = cas_admin_role
my_user.save()

replacing 'cas-dev-three' with whatever your user's first name is. You can check this with the command

[print(”name: “ + user.first_name + “ role: “ + str(user.app_role)) for user in User.objects.all()]

5. in another terminal, navigate to ```/bciers/``` and run ```yarn dev```, and open on localhost:3000 (or 3001 if 3000 is in use by /client/). Log in with the same BCeID account. You should be redirected to a welcome page at localhost:3000/dashboard. Click the link to the facilities page. This should display a wall of text with the operator data for this user.
